src/gatsby-theme-landing-page/styles/variables.module.css

```diff
@@ -0,0 +1,44 @@
+/* CSS custom properties (variables) for customization
+ *
+ * This file shadows the theme file
+ */
+.root {
+  /* typography */
+  --font: "Inter", system-ui, sans-serif;
```

Comment:
Looking good so far. We'll also need to load the webfont since this isn't a system one anymore. Think that could be done in the starter's layout (or via plugin)

Reply:
This will also likely affect the lighthouse perf score, so we should check to see what's best in that regard

Reply:
This might be the way to go now? https://github.com/fontsource/fontsource/tree/main/packages/inter